src/Finder.php

```diff
@@ -654,6 +654,92 @@ public function listFeatures($serviceName = '')
     }
 
     /**
+     * Get the path to the passed model.
+     *
+     * @param string $model
+     * @return string
+     */
+    public function findModelPath($model)
+    {
+        return $this->getSourceDirectoryName() . '/Data/' . $model . '.php';
+    }
+
+    /**
+     * Get the path to the policies directory.
+     *
+     * @return string
+     */
+    public function findPoliciesPath()
+    {
+        return $this->getSourceDirectoryName() . '/Domains/Policies';
```

Comment:
@Mulkave Yeah I totally agree, maybe it's confusing to have them in "Domains" due to its nature. It makes absolute sense! Let's keep them in its own directory: `src/Policies`.